server/graphql/v2/mutation/PersonalTokenMutations.ts

```diff
@@ -74,9 +74,19 @@ const updatePersonalToken = {
       throw new NotFound(`Personal token not found`);
     } else if (!req.remoteUser.isAdminOfCollective(personalToken.collective)) {
       throw new Forbidden('Authenticated user is not the token owner.');
+    } else if (
+      !isUndefined(args.personalToken.preAuthorize2FA) &&
+      args.personalToken.preAuthorize2FA !== personalToken.preAuthorize2FA &&
+      req.personalToken
+    ) {
+      throw new Error(
+        'You cannot change the preAuthorize2FA value of the token you are using. Please use the interface to change it.',
+      );
```

Comment:
Why not something more generic like: no update to a personalToken when authenticating with a personalToken?